content/applications/services/helpdesk/overview/stages.rst

```diff
@@ -0,0 +1,113 @@
+======
+Stages
+======
+
+*Stages* are used to organize the *Helpdesk* pipeline and track the progress of tickets. Stages are
+customizable, and can be renamed to fit the needs of each team.
+
+Create or modify stages
+=======================
+
+.. important::
+   :ref:`Developer mode <developer-mode>` **must** be activated to access the stages menu. To
+   activate developer mode, go to :menuselection:`Settings app --> General Settings --> Developer
+   Tools`, and click :guilabel:`Activate the developer mode`.
+
+To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk app --> Configuration -->
+Stages`.
+
+The default list view on the :guilabel:`Stages` page displays the stages currently available in the
+*Helpdesk* app. They are listed in the order they appear in the pipeline.
+
+To change the order of the stages, click the :icon:`oi-draggable` :guilabel:`(draggable)` icon, to
+the left of the stage name, and drag it to the desired place on the list.
+
+.. image:: stages/stages-list-buttons.png
+   :alt: View of the stage list page emphasizing the buttons used to change the order the stages
+         appear in the list.
+
+.. tip::
+   Change the stage order on the Kanban view of a *Helpdesk* team's pipeline by dragging and
+   dropping individual columns.
+
+To create a new stage, click the :guilabel:`New` button at the top-left of the stage list. Doing so
+reveals a blank stage form.
+
+Choose a :guilabel:`Name` for the new stage, and add a description, if desired.
+
+.. image:: stages/new-stage-details.png
+   :alt: View of a stage's settings page in Odoo Helpdesk.
+
+Add email and SMS templates to stages
+=====================================
+
+When an :guilabel:`Email Template` is added to a stage, an email is automatically sent to the
+customer when a ticket reaches that specific stage in the pipeline. Likewise, adding an
+:guilabel:`SMS Template` triggers an SMS text message to send to the customer.
+
+.. important::
+   SMS Text Messaging is an :doc:`In-App Purchase (IAP) </applications/essentials/in_app_purchase/>`
+   service that requires prepaid credits to work. Refer to `SMS Pricing FAQ
+   <https://iap-services.odoo.com/iap/sms/pricing>`_ for additional information.
+
+To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
+the :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon to the right of the field to edit the
+chosen template.
+
+To create a new template, click the field, and enter a title for the new template. Then, select
+:guilabel:`Create and edit` from the drop-down menu that appears, and complete the form details.
+
+Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
+
+.. image:: stages/sms-template.png
+   :alt: View of an SMS template setup page in Odoo Helpdesk.
+
+.. seealso::
+   :doc:`/applications/general/companies/email_template`
+
+Assign stages to a team
+=======================
```

Comment:
Before you get to this, do you want to mention the 'Sequence' field and what it represents

Reply:
As a rule I usually leave out the "sequence" field when it is mentioned because it is a technical field, and in general people shouldn't need to edit it.